### PR TITLE
RUN: Add implicit build arguments to the beginning of the command line

### PR DESCRIPTION
**Pull request by mchernyavsky**
Fixes https://github.com/intellij-rust/intellij-rust/issues/6802.

changelog: Add implicit build arguments to the beginning of the command line

### CODE REVIEW

File: src/test/kotlin/org/rustSlowTests/ide/actions/RsBuildActionTest.kt [hunk 1]
```diff
@@ -31,41 +32,51 @@ class RsBuildActionTest : CargoBuildTest() {
 
     fun `test build first action`() {
         val testProject = fileTree {
-            toml("Cargo.toml", """
+            toml(
```
Comment (Undin) [inline src/test/kotlin/org/rustSlowTests/ide/actions/RsBuildActionTest.kt:35]: I think you don't want to format this code

Comment (mchernyavsky) [reply src/test/kotlin/org/rustSlowTests/ide/actions/RsBuildActionTest.kt]: Fixed.

**Comment by Undin** [comment]
@mchernyavsky tests fail

**Review by Undin — APPROVED** [review]
bors r+

**Comment by Undin** [comment]
Note, the current changelog note is about what exactly you did, but it's supposed to be about some useful changes from user-side

**Pull request merged**